bucket/gsudo.json

```diff
@@ -2,29 +2,29 @@
     "version": "2.4.4",
     "description": "A Sudo for Windows",
     "homepage": "https://gerardog.github.io/gsudo",
-    "license": "MIT",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/gerardog/gsudo/blob/master/LICENSE.txt"
+    },
```

Comment:
No need to expand `license` and add a `url` because the license is a standard SPDX license and unmodified.